sklearn/tree/_utils.pyx

```diff
@@ -611,9 +597,9 @@ cdef class WeightedMedianCalculator:
                                                   original_median)
         return return_value
 
-    cdef int update_median_parameters_post_remove(self, DOUBLE_t data,
-                                                  DOUBLE_t weight,
-                                                  double original_median) nogil:
```

Comment:
Fair enough then. I would be in favor of not enforcing the 79 limit too strictly for such cases but I don't mind keeping your change.